recipes-support/swupdate/swupdate/swupdate.service

```diff
@@ -2,9 +2,10 @@
 Description=SWUpdate daemon
 Documentation=https://github.com/sbabic/swupdate
 Documentation=https://sbabic.github.io/swupdate
-After=swu-environment.service
+After=swu-environment.service swupdate.socket
```

Comment:
swupdate.socket is required only if we rely on socket-based 
activation (not the case in this layer).
In fact, I would prefer to keep a "minimal" configuration here, 
and let the final user decide.

Reply:
Ah okay, so i remove the install socket and just add swupdate.socket ?

Reply:
Just drop the stuff related to swupdate.socket and keep
only the fix about Type=notify.

So, please rebase with a new commit log which explain
why we have to set this specific Type (see https://www.freedesktop.org/software/systemd/man/systemd.service.html)